test/model/tools/model_test/main.cpp

```diff
@@ -283,75 +284,60 @@ load_workload(const char *file)
 }
 
 /*
- * reduce_counterexample --
- *     Try to find a smaller workload that reproduces a failure.
+ * reduce_counterexample_context_t --
+ *     The context for counterexample reduction.
  */
-static void
-reduce_counterexample(std::shared_ptr<model::kv_workload> workload, const std::string &main_home,
-  const std::string &reduce_home, const std::string &conn_config, const std::string &table_config)
-{
+struct reduce_counterexample_context_t {
+    const std::string &conn_config;
```

Comment:
Is that expected to declare those fields with `&`?

Reply:
I was over-optimizing; you're right that making copies would be less error-prone. Fixed in [4422589](https://github.com/wiredtiger/wiredtiger/pull/10818/commits/4422589bfaf4bea307481f9d6f06336bfc51ddb2).